deploy/crds/capabilities.3scale.net_products_crd.yaml

```diff
@@ -80,7 +80,6 @@ spec:
                           type: string
                         value:
                           description: Limit Value
-                          format: int64
```

Comment:
I guess this change is because of the incompatibility with OpenAPI validation. An alternative to allow big numbers would be using a string and parsing the value (it was done this way with another similar attribute with other Capabilities kind).
Are the limits potentially big? If so we should consider it

Reply:
This is  because 3scale go client objects define value as `int`. https://github.com/3scale/3scale-porta-go-client/blob/86274104320b597755af595f27be8771c944a579/client/types.go#L576

We could change in 3scale porta go client and in the CRD, but in 64bit systems, int range is  [-2^63, 2^63-1]. Really big.

There is no need to do so.